Backend/Core/Services/AuthenticationService.cs

```diff
@@ -115,7 +114,7 @@ public async Task ChangePasswordAsync(ChangePasswordDTO changePasswordDTO, strin
             if (changePasswordDTO.CurrentPassword == changePasswordDTO.NewPassword)
             {
                 throw new HttpException(
-                        ErrorMessages.NewInfoSamePrevious,
+                        ErrorMessages.NewPasswordSameCurrentPassword,
```

Comment:
I don't like the two options, but if I had to choose between the two, the first one is better